source/agora/common/TransactionPool.d

```diff
@@ -197,20 +197,38 @@ public class TransactionPool
 
     ***************************************************************************/
 
-    public int opApply (scope int delegate(Hash hash, Transaction tx) dg) @trusted
+    public int opApply (DG1T) (scope DG1T dg)
```

Comment:
Why the name `DG1T`?

Reply:
Originally because it was the version with a single parameter (as opposed to `DG2T`).

Reply:
I think it can be just `DGT` now.

Reply:
Yes